recipes/topas/build.sh

```diff
@@ -4,7 +4,7 @@ set -eu -o pipefail
 outdir=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
 mkdir -p $outdir
 mkdir -p $PREFIX/bin
-cp -R $SRC_DIR/executable/TOPAS.jar  $outdir/
+cp -R $SRC_DIR/TOPAS-${PKG_VERSION}.jar  $outdir/
```

Comment:
If you write 
```
cp -R $SRC_DIR/TOPAS-${PKG_VERSION}.jar  $outdir/TOPAS.har
```

Reply:
Actually, the `-R` is not needed - it's just one file and we don't want recursion here.